### PR TITLE
Populate readthedocs and add Google Colab notebooks

### PR DESCRIPTION
**Pull request by ivana-truong**
- move most of content from README to readthedocs
- shorten README content
- add links to a Google Colab version of Jupyter notebooks

### CODE REVIEW

File: docs/additional_examples.md [hunk 1]
```diff
@@ -0,0 +1,212 @@
...
+    }
+    exptl {
+      method
+    }
+  }
+}
+```
+```python
+from rcsbapi.data import Query
+query = Query(input_ids={"entry_ids": ["1STP","2JEF","1CDG"]},input_type="entries", return_data_list=["entries.rcsb_id", "struct.title", "exptl.method"])
```
Comment (piehld) [inline docs/additional_examples.md:21]: For this and other examples, I would suggest cleaning up the indenting so it's all visible without scrolling left to right:

```suggestion
query = Query(
    input_ids={"entry_ids": ["1STP", "2JEF", "1CDG"]},
    input_type="entries",
    return_data_list=["entries.rcsb_id", "struct.title", "exptl.method"]
)
```

File: docs/additional_examples.md [hunk 2]
```diff
@@ -0,0 +1,212 @@
...
+  entries(entry_ids: ["1STP", "2JEF", "1CDG"]) {
+    rcsb_id
+    struct {
+      title
+    }
+    exptl {
+      method
+    }
+  }
+}
```
Comment (piehld) [inline docs/additional_examples.md:17]: Also, I would recommend moving the GraphQL query to _after_ the Python examples. You could say something like, 

"Fetch information about structure title and experimental method for PDB entries:"
<python example>
"The above code performs the same query as the following GraphQL query:"
<GraphQL query>

File: docs/query_construction.md [hunk 3]
```diff
@@ -0,0 +1,203 @@
+# Query Construction
+
+## Query Objects
+Constructing a query object requires three inputs. The JSON response to a query is stored in the `response` attribute of a Query object and can be accessed using the `get_response()` method.
+```python
+from rcsbapi.data import Query
+
+# constructing the Query object
+query = Query(input_ids={"entry_id":"4HHB"},input_type="entry", return_data_list=["exptl.method"])
```
Comment (piehld) [inline docs/query_construction.md:9]:
```suggestion
query = Query(input_ids={"entry_id":"4HHB"}, input_type="entry", return_data_list=["exptl.method"])
```

File: docs/query_construction.md [hunk 4]
```diff
@@ -0,0 +1,203 @@
...
+
+## Helpful Methods
+There are several methods included to make working with query objects easier. These methods can help you refine your queries to request exactly and only what you want and further understand the GraphQL syntax.
+
+### get_editor_link()
+This method returns the link to a [GraphiQL](https://data.rcsb.org/graphql/index.html) window with the query. From the window, you can use the user interface to explore other fields and refine your query. Method of Query class.
+
+```python
+from rcsbapi.data import Query
+query = Query(input_ids={"entry_id":"4HHB"},input_type="entry", return_data_list=["exptl"])
```
Comment (piehld) [inline docs/query_construction.md:134]:
```suggestion
query = Query(input_ids={"entry_id":"4HHB"}, input_type="entry", return_data_list=["exptl"])
```

File: docs/query_construction.md [hunk 5]
```diff
@@ -0,0 +1,203 @@
...
+
+## Trouble-shooting
+### ValueError: Not a unique field
+Some fields are redundant within our GraphQL Data API schema. For example, "id" appears over 50 times. To allow for specific querying, redundant fields are identified by the syntax `<type>.<field name>`. If you request a redundant field without this syntax, a `ValueError` will be returned stating that the field exists, but is redundant. You can then use `get_unique_fields("<field name>")` to find notation that would specify a unique field for the given name.
+
+```python
+from rcsbapi.data import Query
+
+# querying a redundant field
+query = Query(input_ids={"entry_id":"4HHB"},input_type="entry", return_data_list=["id"])
```
Comment (piehld) [inline docs/query_construction.md:172]:
```suggestion
query = Query(input_ids={"entry_id":"4HHB"}, input_type="entry", return_data_list=["id"])
```

**Review by piehld — COMMENTED** [review]
Thanks @ivana-truong! This documentation looks fantastic. I just had a couple minor formatting suggestions that I think would be good to update throughout. 

One more general question I have is whether you know why/if the API Documentation doesn't display the tool methods etc. as it does for the search API package (e.g., https://rcsbsearchapi.readthedocs.io/en/latest/api.html)? I wonder if that's what the `api.rst` file was doing?

**Comment by ivana-truong** [comment]
@piehld Thanks for the review!
I made the requested changes and I like how it looks without the side scrolling boxes. 

The api.rst file never displayed anything on the readthedocs page, so I got rid of it, but looking into it further, it is supposed to generate the documentation that you linked. I restored it and have been playing around with it, but I can't get it to work. When I build the html file locally, it looks correct, but I just can't get it to display on the readthedocs. 

I'm going to continue tinkering with it and seeing if I can get the api.rst file to work. The other changes have been made if you would like to check them!

**Pull request merged**